### DISCUSSION TITLE
Preview pull-request contents in Dry-Run mode

### DISCUSSION THREAD

**Discussion by ricochet1k** [renovatebot/renovate#23435] [Suggest an Idea]
### Tell us more.

When in dry-run mode I'd like to be able to preview what pull requests it wants to create/modify/etc.

I have made an attempt at making this change here: https://github.com/ricochet1k/renovate/tree/dry-run-preview
I've moved dry-run check/log logic into it's own file and added file writing to it.

**Comment by rarkins** [comment]
This should be done within the logs only. We don't want to add extra file writing
